backend/db/models.py

```diff
@@ -176,11 +176,11 @@ def __repr__(self) -> str:
 
 
 def validate_extractor_owner(
```

Comment:
Why conceptually should this one change?

Reply:
On mobile so I don't see the code -- probably shouldn't I did a sed command and maybe missed some spots